CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_zendesk_source v0.7.0
+## 🎉 Documentation and Feature Updates 🎉
+- Updated README documentation for easier navigation and dbt package setup.
+- Included the `zendesk_[source_table_name]_identifier` variables for easier flexibility of the package models to refer to differently named sources tables.
+- Databricks compatibility 🧱 ([#29](https://github.com/fivetran/dbt_zendesk_source/pull/29))
+- README clarifications on installing the source and transform Zendesk packages together.
```

Comment:
Could we probably consolidate this with the first bullet?

Reply:
yep